ccmain/thresholder.cpp

```diff
@@ -312,7 +312,7 @@ void ImageThresholder::ThresholdRectToPix(Pix* src_pix,
       bool white_result = true;
       for (int ch = 0; ch < num_channels; ++ch) {
         int pixel = GET_DATA_BYTE(const_cast<void*>(
```

Comment:
I think that both casts here can be removed. This also applies to all other instances of GET_DATA_BYTE. I suggest to fix that in a separate pull request (easier review).